tock/projects/templatetags/project_tags.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+register = template.Library()
+
+
+def get(value, key):
+    return value.get(key)
+
+
+def default(value, default):
```

Comment:
This is random but looks like a default function was [working](https://github.com/jmcarp/tock/blob/project-data-table/tock/tock/templates/projects/project_detail.html#L7) before you added this.